File: Python/Series/Series.py
#Function to find the factorial of a number
def factorial(num):                                        
    if num==1:
        return num
    else:
        return float(num*factorial(num-1))

print("Series: 1/1! + 2/2! + 3/3! + ... + N/N!\n")
terms=input("Enter the number of terms to find the sum:")
i=1.0
sum_terms=0.0  

#Finding the sum of the series
while(i<=terms):                                           
    facto=factorial(i)
    sum_terms+=i/facto
    i=i+1
print "Sum:"
print(sum_terms) 

File: C/Series/Series.c
#include<stdio.h>

//Finding factorial of a number
float factorial(int num){
    if(num == 1)
        return num;
    else
        return num*factorial(num-1);
}

int main(){
    int terms,i;
    float facto,sum=0.0;
    printf("Series: 1/1! + 2/2! + 3/3! + ... + N/N!\nEnter the number of terms to find sum: ");
    scanf("%d",&terms);

    //Finding the sum of the series
    for(i=1;i<=terms;i++){
        facto=factorial(i);
        sum+=i/facto;
    }
    printf("Sum = %f\n",sum);
}